subsys/net/lib/sntp/Kconfig

```diff
@@ -9,6 +9,13 @@ menuconfig SNTP
 
 if SNTP
 
+config SNTP_UNCERTAINTY
+	bool "Calculate SNTP uncertainty and set uptime timestamp"
+	help
+	  Calculate SNTP time uncertainty, and supply the uncertainty and
+	  uptime timestamp in the sntp_time struct. Time is then given as:
+	  seconds,fraction +/- uncertainty at uptime timestamp.
+
```

Comment:
Could there be some more description why user would want to enable this? Also wondering should this be enabled by default, WDYT?

Reply:
I don't think it should be enabled by default, as it uses extra resources, and this feature is only necessary for applications where time certainty is critical.

What do you think for this description:
```
	  Get a more reliable timestamp by supplying the sntp_time struct with an
	  uptime timestamp of when the time was valid and an uncertainty of the 
	  time in microseconds. Time is then given as: 
	  seconds,fraction +/- uncertainty at uptime timestamp.
```